### ISSUE TITLE
Date interpretted as Unix File Path

### ISSUE THREAD

**Issue by frakman1** [bensadeh/tailspin#69]
On MacOS 13.1, using iTerm.
tailspin version 1.6.1

I use a black background in my terminal window and the blue in the time format was hard to read. I changed it to `yellow` in my `~/.config/tailspin/config.toml` file as such:
```
...
[time]
time = { fg = "yellow" }
zone = { fg = "red" }
# To shorten the time, uncomment the line below
# shorten = { to = "␣", style = { fg = "blue" } }
```

Anyway, when I tried to view this log file, part of the date appears to be interpretted as a unix file path.
Also, I think the logic expects a `.` after the time but instead is seeing a `:` and interpretting it as something else.

<img width="659" alt="image" src="https://github.com/bensadeh/tailspin/assets/5826484/31e30188-8a91-486c-890e-4108aa8ead51">

```
$cat "com.flexibits.fantastical2.mac 2023-10-02--15-42-31-759.log"
2023/10/02 11:42:31:738  Location manager failed: Error Domain=kCLErrorDomain Code=0 "(null)"
2023/10/02 11:43:27:698  Location manager error: Error Domain=kCLErrorDomain Code=0 "(null)"
2023/10/02 11:47:39:850  Location manager failed: Error Domain=kCLErrorDomain Code=0 "(null)"
```

**Comment by bensadeh** [comment]
Thanks for opening this issue up with screenshots and detailed description to reproduce.

Unfortunately, the dates of the format yyyy/mm/dd are not currently supported. In this case, I think it would solve the issue because it would highlight the dates before the unix path highlighter got a chance to look at it.

I would like to add support for this, however it is a bit tricky because the regexes to highlight the dates are so different, they can't be reused.

I can't give a timeline to this yet, but support for this format is something that should be added at some point for sure.

For now, what locale are you using, if you don't mind me asking?

**Comment by frakman1** [comment]
No problem. Happy to help.

I look forward to testing the next update.

I am in North America, EST if that helps.

**Comment by bensadeh** [comment]
Fixed in `3.0.0` 🎉
